Gopkg.lock

```diff
@@ -44,6 +44,12 @@
   revision = "b26d9c308763d68093482582cea63d69be07a0f0"
   version = "v0.3.0"
 
+[[projects]]
+  name = "github.com/DataDog/datadog-go"
```

Comment:
you can't change that without a PR for vendor first

Reply:
@ldemailly sure thing, what do I need to do for this?

Reply:
https://github.com/istio/vendor-istio/pull/1

Reply:
merged, now update to master in vendor and update this pr (git diff should show vendor change without -dirty)

thanks!

Reply:
I've not worked with submodules before so this is still new to me. I've checked-out/pulled master now in `./vendor` which is up to date. Git status for this branch now shows
```
Changes not staged for commit:
  (use "git add <file>..." to update what will be committed)
  (use "git checkout -- <file>..." to discard changes in working directory)

	modified:   vendor (new commits)
```

Do I commit this and then push?

Reply:
yes to pickup your now part of vendor's master in this PR